software/openuem.yml

```diff
@@ -0,0 +1,14 @@
+name: OpenUEM
+website_url: https://openuem.eu
+source_code_url: "https://github.com/open-uem"
+description: "Unified endpoint manager which keeps an inventory of your IT assets reported by agents installed on your endpoints with a clean and concise UI"
```

Comment:
```suggestion
description: "Unified endpoint manager which keeps an inventory of your IT assets reported by agents installed on your endpoints with a clean and concise UI."
```

Reply:
Hi, I've applied the change you suggest. Thank you